OpenSourceOrg/dotOrg
# Event Manager updates (dev)

#### Changes proposed in this Pull Request

* Style booking button
* Don't show timezone if it's an "all day" event

#### Testing instructions

* Open an event page (like [this](https://open-source-initiative-development.mystagingwebsite.com/events/open-source-ai-definition-town-hall-may-31)) and confirm that the "Submit Booking" button is styled as the "Join us" button
* If the event is "all day", confirm that the `#_EVENTTIMEZONE` placeholder isn't displaying anything. 

Mentions https://github.com/a8cteam51/opensource-net/issues/49
